hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -1466,8 +1466,6 @@ protected final HoodieTable initTable(WriteOperationType operationType, Option<S
 
     // Validate table properties
     metaClient.validateTableProperties(config.getProps());
-    // Make sure that FS View is in sync
-    table.getHoodieView().sync();
```

Comment:
@alexeykudinkin : I see you have added this with https://github.com/apache/hudi/pull/4739 Do you happened to know the reason?

Reply:
This has been carried over from the Spark client